.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "Library/ShadowPath/ShadowPath/shadowsocks-libev"]
 	path = Library/ShadowPath/ShadowPath/shadowsocks-libev
-	url = git@github.com:icodesign/shadowsocks-libev.git
+	url = Library/ShadowPath/ShadowPath/shadowsocks-libev.git
```

Comment:
Something wrong here. It should be remote github url, not the local one.

Reply:
Thank you, I will fix it.